docs/ardana/testing.md

```diff
@@ -210,6 +210,91 @@ prepare_infra
 ...
 ```
 
+## Custom Ardana Gerrit CI builds
+
+The default integration test job launched by the Gerrit Jenkins CI uses
+a `standard` generated input model, with an integrated deployer, 2
+controller nodes and one SLES compute node. It also runs against a
+`develcloud` cloud media, on top of which it applies the most recent
+Ardana ansible changes and, of course, the target Gerrit change under
+test and all its Gerrit change dependencies.
+
+This is sufficient for most cases, but sometimes there are special
+circumstances that need to be supported. For example:
+
+* the target Gerrit change may need to be tested against a different
+input model scenario or cloud configuration. Some examples of this are:
+
+  * a Gerrit change that impacts the RHEL compute node support also needs
+  to be tested against a cloud input model that includes RHEL (CentOS)
+  compute nodes.
+  * a Gerrit change that touches on functionality that can only be tested
+  on bare-metal deployments (e.g. cobbler) needs to be tested against
+  a QA bare-metal environment in addition, or instead of a virtual one
+
+* the target Gerrit change implements a new piece of functionality for
+which new packages also need to be installed in addition to using a
+different cloud configuration
+
+* the target Gerrit change implements a new piece of functionality for
+which CI coverage is still being developed, but available in a separate
+automation repository fork or branch
+
+* ultimately, the Gerrit change might not require integration testing
+at all, for example because the code changes do not affect any functionality
+areas (e.g. documentation or git configuration changes). For this cases,
+the CI should support skipping the integration testing altogether.
+However, this feature should not be abused.
+
+The Ardana Gerrit CI supports all these use-cases by allowing the users
```

Comment:
So just to summarise this is really just a simplified build parameter interface to submitting a custom openstack-ardana job to test a gerrit change?

Reply:
Precisely ! I updated the docs to make that clearer